web/src/main/webapp/WEB-INF/pages/candidate_create.jsp

```diff
@@ -109,11 +109,9 @@
                                 <input type="text" class="input form-control" name="email" id="email" data-bind="valueWithInit: 'email'"
                                        value="${candidate.email}"
                                        placeholder="${i18nemail}"
-                                       data-error="<spring:message
-                                    code="candidate.error.email.empty"/>"
+                                       <spring:message code="candidate.error.email.empty" var="messValue"/>data-error="${messValue}"
```

Comment:
The `<spring:message>` tag is now inside the `<input>` start tag, which does not conform the xml nor the html standards.
Move the `<spring:message>` tag before the `<input>` tag.